frontend/components/ThemeProvider/Theme.ts

```diff
@@ -0,0 +1,160 @@
+interface IGeneralProperty {
+  body: string;
+  text: string;
+}
+
+interface IButton<T> {
+  default: T;
+  hover: T;
+}
+
+interface IStatusButton<T> {
+  success: T;
+  warning: T;
+  danger: T;
+}
+
+export interface ITheme {
+  body: string;
+  text: string;
+  border: string;
+  svg: {
+    fill: string;
+  };
+  section: {
+    body: string;
+    text: string;
+  };
+  button: IStatusButton<IButton<IGeneralProperty>>;
+}
+
+const primary = {
+  blue: "#581B98",
+  purple: "#9C1DE7",
+  pink: "#F3558E",
+  yellow: "#F3558E",
+  grey: "#AF9FC0",
+};
+
+const neutral = {
```

Comment:
These are constants for the colours, let's shove them in a constants file dedicated to colour constants at root level :P

Reply:
Agreed 👍

Reply:
we can do that.